2.2/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM openshift/base-centos7
+
+# This image provides a Ruby 2.2 environment you can use to run your Ruby
+# applications.
+
+MAINTAINER Jakub Hadvig <jhadvig@redhat.com>
```

Comment:
+1 to use mailing list here (I think that is what you do already @hhorak right?)